<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Tikas-MacBook-Air.local" failures="2" tests="2" name="Test(failed)" time="0.092" errors="0" timestamp="2024-04-16T14:36:01 PDT">
  <testcase classname="testNG_Scripts.Test_Scenario3" name="testcase1" time="0.02">
    <failure type="java.lang.AssertionError" message="expected [Good Evening] but found [Good Morning]">
      <![CDATA[java.lang.AssertionError: expected [Good Evening] but found [Good Morning]
at testNG_Scripts.Test_Scenario3.testcase1(Test_Scenario3.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testcase1 -->
  <testcase classname="testNG_Scripts.Test_Scenario5" name="testcase1" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [Hello] but found [Hi]">
      <![CDATA[java.lang.AssertionError: expected [Hello] but found [Hi]
at testNG_Scripts.Test_Scenario5.testcase1(Test_Scenario5.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testcase1 -->
</testsuite> <!-- Test(failed) -->
